api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -187,6 +188,11 @@
         "special": lambda x: x == "true",
         "column": offers_models.Offer.is_headline_offer,
     },
+    "HIGHLIGHT_REQUEST": {
+        "field": "highlight",
+        "column": highlights_models.HighlightRequest.highlightId,
+        "inner_join": "highlight_request",
```

Comment:
Sans tester, mon intuition pressent que ça va marcher pour l’opérateur IN mais pas pour le NOT_IN.